bindings/lua/tests/run.sh

```diff
@@ -14,4 +14,4 @@ fi
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-lua $DIR/test_string.lua
+# lua $DIR/test_that_requires_yarpserver.lua
```

Comment:
Is this a leftover?

Reply:
Maybe add a comment, i.e.
```diff
-lua $DIR/test_string.lua
+# Any test that requires yarpserver would go here" 
+# lua $DIR/test_that_requires_yarpserver.lua

Reply:
Sure, added at 8b7aa10d720c17d08aa7b49fac45765f84a02fd9 : )